pom.xml

```diff
@@ -7,6 +7,28 @@
     <groupId>org.example</groupId>
     <artifactId>employment-system</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <dependencies>
+        <dependency>
+            <groupId>org.openjfx</groupId>
+            <artifactId>javafx-graphics</artifactId>
+            <version>11.0.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.openjfx</groupId>
+            <artifactId>javafx-fxml</artifactId>
+            <version>11.0.1</version>
+        </dependency>
+        <dependency>
```

Comment:
I do not think that we need this dependency here.

Reply:
I think that we need that.